src/style.css

```diff
@@ -1 +1,118 @@
 /* empty source */
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+}
+
+.card_image {
+  margin: 0 auto 40px;
+  display: block;
+  height: 134px;
+  width: 160px;
+}
+
+.card_product_name {
+  display: block;
+  margin: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card_product_code {
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card_review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin: 0;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.starts-star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars--1 .starts-star:nth-child(-n+1),
+.stars--2 .starts-star:nth-child(-n+2),
+.stars--3 .starts-star:nth-child(-n+3),
+.stars--4 .starts-star:nth-child(-n+4),
+.stars--5 .starts-star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card_review_text {
+  color: #060b35;
+}
+
+.card_price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  line-height: 18px;
+}
+
+.card_price_text {
+  font-size: 12px;
+  color: #616070;
+}
+
+.card_price_money {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card_buy_button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  box-sizing: border-box;
+  padding: 12px 0;
+  height: 40px;
+  width: 166px;
+
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  text-transform: uppercase;
+  text-align: center;
+  text-decoration: none;
+
+  color: #fff;
+  background-color: #00acdc;
+  border: 2px solid #00acdc;

```

Comment:
@Edriss572 why you disregard your group-mate comments?
they are reasonable
create a variable for repeating colors

Reply:
i just saw it i will do it as soon as possible